static/src/stylesheets/pasteup/forms/_styles.scss

```diff
@@ -247,10 +247,18 @@
     width: 13px;
 }
 
+$delete-warn-red: #b20a0a;
```

Comment:
Could you choose from [one of the many reds](http://guardian.github.io/pasteup-palette/demo/) we already have, instead of intruducing another variable, please? We are trying not to add any new colors ...